src/main/java/com/thohol/keycloak/JsonGraphQlRemoteClaim.java

```diff
@@ -38,19 +41,12 @@
     private final static String REMOTE_GRAPHQL = "remote.graphql";
     private final static String REMOTE_GRAPHQL_QUERY = "remote.graphql.query";
     private final static String REMOTE_GRAPHQL_RESULT_PATH = "remote.graphql.path";
-
-
+    private final static String RETRY_REQUEST = "retry.request";
```

Comment:
not really important but `"remote.request.retry"` would be more consistent

```suggestion
    private final static String RETRY_REQUEST = "remote.request.retry";
```